{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/careers/models.py

```diff
@@ -37,6 +37,50 @@ def select_closed(self):
         )
 
 
+class CareerLocation(models.Model):
+
+    title = models.CharField(
+        max_length=100,
+    )
+
+    street_address = models.CharField(
+        max_length=128,
+        blank=True,
+        null=True,
+    )
+
+    city = models.CharField(
+        max_length=128,
+        blank=True,
+        null=True,
+    )
+
+    region = models.CharField(
+        max_length=128,
+        blank=True,
+        null=True,
+    )
+
+    postcode = models.CharField(
+        max_length=128,
+        blank=True,
+        null=True,
+    )
+
+    country = models.CharField(
+        max_length=2,
+        blank=True,
+        null=True,
+        help_text="The country's ISO ALPHA-2 code"
+    )
+
+    class Meta:
```

Comment:
Use a `verbose_name` of 'location' here - the 'career' part of the name is redundant.